priv/templates/bin

```diff
@@ -11,30 +11,30 @@ REL_DIR="$RELEASE_ROOT_DIR/releases/$REL_VSN"
 ERL_OPTS="{{ erl_opts }}"
 
 find_erts_dir() {
-    local erts_dir="$RELEASE_ROOT_DIR/erts-$ERTS_VSN"
-    if [ -d "$erts_dir" ]; then
-        ERTS_DIR="$erts_dir";
+    l_erts_dir="$RELEASE_ROOT_DIR/erts-$ERTS_VSN"
```

Comment:
Just my personal preference here, but I think using initials in variables and functions confusing. How about prepending the variable name with one or two underscores instead?

Actually you could also just leave the name as is. I believe it was made lowercase originally because it was a local variable and only mentioned in this function. `erts_dir` is also used in the `find_erts_dir` function, but it's also reassigned, so there shouldn't be any conflict.